resources/smcp-templates/v2.6/base

```diff
@@ -34,7 +34,7 @@ spec:
 
   security:
     identity:
-      type: Kubernetes
+      type: ThirdParty
```

Comment:
This change will break existing meshes if a control plane is updated from an older version to 2.6.  Existing proxies will not be able to update their serving certificates for mtls until they are reinjected.